lighthouse-core/audits/critical-request-chains.js

```diff
@@ -42,15 +42,15 @@ class CriticalRequestChains extends Audit {
     };
   }
 
-  /** @typedef {{depth: number, id: string, chainDuration: number, chainTransferSize: number, node: LH.Audit.SimpleCriticalRequestNode[string]}} CrcNodeInfo */
+  /** @typedef {{depth: number, id: string, chainDuration: number, chainTransferSize: number, node: LH.Audit.Details.SimpleCriticalRequestNode[string]}} CrcNodeInfo */
 
   /**
-   * @param {LH.Audit.SimpleCriticalRequestNode} tree
+   * @param {LH.Audit.Details.SimpleCriticalRequestNode} tree
```

Comment:
going through all of our `d.ts` files, this was the only type change that needed to happen in a JS file. `SimpleCriticalRequestNode` really is an audit details type, it just predates the file `audit-details.d.ts`. Moving it to audit-details avoids a circular dependency of `audit-details.d.ts` back to `SimpleCriticalRequestNode` in `audit.d.ts`.